bigdata/src/main/scala/org/w3/banana/bigdata/Bigdata.scala

```diff
@@ -0,0 +1,30 @@
+package org.w3.banana.bigdata
```

Comment:
we have started breaking the package declarations in two:

```
package org.w3.banana
package bigdata
```

Not everything has been ported yet (far from it) but the new code should adopt that approach. It simplifies imports.